### PR TITLE
[dr_libs] New library

### PR DESCRIPTION
**Pull request by fcorso2016**
Specify library name and version:  **dr_libs/0.13.10**

Pulled in a public domain library that consists of 3 header-only files that are used to ingest and re-encode audio data between different formats.

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
@fcorso2016 Thank you for your contribution, please, don't forget to ask for access request on the issue #4, otherwise, the CI will not build your PR.

**Review by uilianries — COMMENTED** [review]
Thank you for your contribution, please, don't forget to add a test package to validate your new recipe. There is a good template here: https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/header_only/all/test_package

File: recipes/dr_libs/all/test_package/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_DrLibs LANGUAGES CXX) # if the project uses c++
+
+find_package(dr_libs REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE dr_libs::dr_libs)
```
Comment (uilianries) [inline recipes/dr_libs/all/test_package/CMakeLists.txt:7]:
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE dr_libs::dr_libs)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

File: recipes/dr_libs/all/test_package/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.15)
+project(test_DrLibs LANGUAGES CXX)
+find_package(dr_libs REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE dr_libs::dr_libs)
```
Comment (czoido) [inline recipes/dr_libs/all/test_package/CMakeLists.txt:6]:
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE dr_libs::dr_libs)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```

**Review by czoido — DISMISSED** [review]

**Review by czoido — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**